src/sdk/account/classes/account-total-balances-item.ts

```diff
@@ -0,0 +1,17 @@
+import { Type } from 'class-transformer';
+import { AccountTotalBalancesMetadata } from './account-total-balances-metadata';
+
+export class AccountTotalBalancesItem {
+  chainId: number;
+
+  protocol: string;
+
+  service: string;
+
+  category: string;
```

Comment:
I think we have a type for that on the backend

Reply:
We do have a type but it's not used in the Class. On the backend `category` is `string` as well.